srcs/Transaction/CGIExecutor.cpp

```diff
@@ -88,64 +89,96 @@ static std::string read(int fd) {
             break;
         }
         if (byte == -1) {
-            throw std::runtime_error("Error: read failed " +
-                                     std::string(strerror(errno)));
+            return -1;
         }
         buf[byte] = '\0';
-        ret += std::string(buf);
+        *ret += std::string(buf);
 
         // 読み込んだバイト数が定数より少ない場合は全部読み込んだ
         if (byte < 4096) {
             break;
         }
     }
 
+    return 0;
+}
+// 1000 000
+static int wait_until(pid_t pid, int *st, int sec) {
+    int cnt = 0, ret = 0;
+    int step = 10000;  // 10ms
```

Comment:
```suggestion
    int step_us = 10000;  // 10ms
```

数値を扱うときは変数名に単位を含めると事故が防げる気がします。

Reply:
修正しました！